common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAudience.java

```diff
@@ -154,6 +160,25 @@ public void setTenantId(String tenantId) {
         mTenantId = tenantId;
     }
 
+    public static String getAudienceFromAuthority( final AzureActiveDirectoryAuthority authority) {
+
+        if (authority == null || StringUtil.isNullOrEmpty(authority.getAudience().getTenantId())) {
```

Comment:
Can `authority.getAudience` return null?

Reply:
I didnt come across such use case, but I figured its better to have, It was mentioned in todays scrum to make the Authotiry @NonNull, so I think there is a low probability that authority would be null

Reply:
Not the authority but I'm referring to the audience itself